src/components/legend/draw.js

```diff
@@ -426,7 +448,7 @@ function setupTraceToggle(g, gd) {
         }
     });
     traceToggle.on('mouseup', function() {
-        if(gd._dragged) return;
+        if(gd._dragged || gd._editing) return;
```

Comment:
@alexcjohnson made some valid points above. But as `editable: true` is still a niche feature, I don't it's worth the time to improve its interactions with other components. So, I'm very much a fan of @rpaskowitz's `gd._editing` solution in this PR.

With regards to adding another `gd._` keys: yes I agree, we should try to move away from this pattern in the long run. But with this [test](https://github.com/plotly/plotly.js/blob/0cf2ee101f9f373347aebfaf773fa1c16bc4d19e/test/jasmine/tests/plots_test.js#L385) (that @rpaskowitz will need to patch :wink: ) keeps us honest.